test/glific/interactive_templates_test.exs

```diff
@@ -504,83 +526,185 @@ defmodule Glific.InteractiveTemplatesTest do
+    """
+
+    {:ok, %{export_data: export_data}} =
+      InteractiveTemplates.export_interactive_template(interactive, add_translation)
+
+    assert String.trim(export_data) == String.trim(expected_export_data)
+  end
+
+  test "export the interactive template when add translation is false",
+       %{organization_id: _organization_id} = attrs do
+    add_translation = false
+
+    interactive =
+      Fixtures.interactive_fixture(Map.merge(@valid_list_attrs, attrs))
+
+    list_export_data = """
+    Attribute,en
+    Title,glific
+    Body,How was your experience with Glific?
+    GlobalButtonTitle,Glific Features
+    ItemTitle 1,Excitement level
+    ItemSubtitle 1,Excitement level
+    OptionTitle 1.1,Great
+    OptionDescription 1.1,Awesome
+    """
+
+    {:ok, %{export_data: export_data}} =
+      InteractiveTemplates.export_interactive_template(interactive, add_translation)
+
+    assert String.trim(export_data) == String.trim(list_export_data)
+
+    # type quick reply with footer
+    interactive =
+      Fixtures.interactive_fixture(Map.merge(@valid_footer_attrs, attrs))
 
     expected_export_data = """
-    id,Attribute,en,hi
-    #{interactive_id},Footer,caption is footer,कैप्शन पाद लेख है
-    #{interactive_id},Header,Glific Features,शानदार विशेषताएं
-    #{interactive_id},Text,How was your experience with Glific?,ग्लिफ़िक त्वरित उत्तर का परीक्षण करें?
-    #{interactive_id},OptionTitle 1,Great,उत्कृष्ट
-    #{interactive_id},OptionTitle 2,Awesome,शानदार
+    Attribute,en
+    Footer,caption is footer
+    Header,Glific Features
+    Text,How was your experience with Glific?
+    OptionTitle 1,Great
+    OptionTitle 2,Awesome
     """
 
     {:ok, %{export_data: export_data}} =
-      InteractiveTemplates.export_interactive_template(interactive)
+      InteractiveTemplates.export_interactive_template(interactive, add_translation)
 
     assert String.trim(export_data) == String.trim(expected_export_data)
   end
+
+  test "import the interactive template",
+       %{organization_id: _organization_id} = attrs do
+    # type list
+    interactive =
+      Fixtures.interactive_fixture(Map.merge(@valid_list_attrs, attrs))
+
+    translated_data =
+      [
+        ["Attribute", "en", "hi"],
+        ["Title", "glific", "ग्लिफ़िक"],
+        ["Body", "How was your experience with Glific?", "ग्लिफ़िक त्वरित उत्तर का परीक्षण करें?"],
+        ["GlobalButtonTitle", "Glific Features", "शानदार विशेषताएं"],
+        ["ItemTitle 1", "Excitement level", "उत्साह का स्तर"],
+        ["ItemSubtitle 1", "Excitement level", "उत्साह का स्तर"],
+        ["OptionTitle 1.1", "Great", "उत्कृष्ट"],
+        ["OptionDescription 1.1", "Awesome", "शानदार"]
+      ]
+
+    {:ok, imported_temp} =
+      InteractiveTemplates.import_interactive_template(translated_data, interactive)
+
+    {:ok, translated_temp} =
+      Repo.fetch_by(Glific.Templates.InteractiveTemplate, %{id: interactive.id})
+
+    assert translated_temp == imported_temp
```

Comment:
Here we are asserting the same data each other. So it will be always true.
Instead we should equate with the "would be" interactive template struct and the newly saved template (after importing) in DB.
So the rhs can be a modified version of `valid_list_attrs` (with the new imported versions, if any)